pkg/ipamplugin/ipam_plugin.go

```diff
@@ -155,6 +155,9 @@ func cmdAdd(args *skel.CmdArgs) error {
 	}
 
 	ctx := context.Background()
+	ctx, cancel := context.WithTimeout(ctx, 120*time.Second)
```

Comment:
I suggested switching the context for the cmdAdd to have a timeout (instead of just on the IPPools call). As I understand it, @rafaelvanoni had to increase the timeout from 10s to 120s to get the tests to pass.
@caseydavenport do you have any idea why there would be that much time spent in here?

@rafaelvanoni Looking back at the build failures, I see in build 4 that it shows many of the failures took 0.0x seconds. Do you know if that build had the shorter timeout?

Reply:
That was either a 10 or 30 sec timeout

Reply:
With the 10 sec timeout again. We've got 1 failure, 2 passes so far. I've got another run going.